packages/components/tests/integration/components/hds/form/checkbox/field-test.js

```diff
@@ -89,10 +89,10 @@ module('Integration | Component | hds/form/checkbox/field', function (hooks) {
   test('it should spread all the attributes passed to the component on the input', async function (assert) {
     assert.expect(3);
     await render(
-      hbs`<Hds::Form::Checkbox::Field checked="checked" class="my-class" data-test1 data-test2="test" />`
```

Comment:
mmm. probably I would have not merged the two tests, for me they're testing two different things. the old one was testing the `...attributes` spread, while the one you've added in theory should test the `@name` argument, but is not.

Reply:
discussed with @alex-ju and he showed me that the `field` is not using `name` as argument, but as attribute. I am going to resolve this thread, and see if/how this test can be adapted to test both things (spreading in general, and `name` in particular).